presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraDistributed.java

```diff
@@ -72,7 +73,10 @@ public void testRenameColumn()
     @Override
     public void testInsert()
     {
-        // Cassandra connector currently does not support insert
+        execute("DROP TABLE IF EXISTS tpch.table_insert");
+        execute("CREATE TABLE tpch.table_insert (id int PRIMARY KEY)");
+        execute("INSERT INTO tpch.table_insert (id) VALUES (1)");
```

Comment:
re testInsert(): great, maybe make a note that you've modified it.

re all supported datatypes: if you look at ``testPartitionKeyPredicate``, ``testSelect``, etc from ``TestCassandraIntegrationSmokeTest``, they already use ``TABLE_ALL_TYPES``, which is created in ``CassandraTestingUtils`` via cql. You can create a new all types table (``createTableAllTypes``) just for insert, which then gets dropped afterwards in a finally block. It's not so bad to use cql to set up the tables, because it ensures that the tables are as they are supposed to be; using Presto both for setup and testing could hide subtle bugs.